imports/plugins/core/orders/client/components/invoice.js

```diff
@@ -1,35 +1,41 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import moment from "moment";
 import { formatPriceString } from "/client/api";
-import { Translation } from "/imports/plugins/core/ui/client/components";
+import { Translation, CardGroup, Card, CardBody, CardHeader } from "/imports/plugins/core/ui/client/components";
```

Comment:
Use `@reactioncommerce/reaction-components` instead of `/imports/plugins/core/ui/client/components`